src/sse/route-service.ts

```diff
@@ -2,13 +2,16 @@ import SSE from 'express-sse';
 import ChainwebEventService from './chainweb-event.js';
 import Logger from './logger.js';
 import ChainwebCutService from './chainweb-cut.js';
-import { TransactionType, InitialEvent } from './types.js';
+import { TransactionType, InitialEvent, ChainwebCutData } from './types.js';
 import config from '../config/index.js';
 import { WIRE_PROTOCOL_VERSION } from '../config/constants.js';
-
-const { network, confirmationDepth, heartbeatInterval } = config;
+import ChainwebDataHeightTracker from './chainweb-data-height-tracker.js';
 
 let cut;
+const heightTracker = new ChainwebDataHeightTracker();
+heightTracker.start();
+
+const { network, confirmationDepth, heartbeatInterval } = config;
```

Comment:
I'm not sure if starting the tracker in the root of the file is best practice. A solution could be to create a singleton getter and do initialisation there.